changelog.json

```diff
@@ -1,5 +1,12 @@
 {
   "releases": {
+    "1.5.0-beta3": [
+      "[Added] Will now reportup when a binary file is in a conflicted state - #6072",
+      "[Fixed] Handle really long banner messages - #6050",
+      "[Fixed] Changes are updated after aborting a merge - #6052",
```

Comment:
:art: the three fixes here are part of the Conflict Resolution flow that users noticed on beta, but we should omit it from the production release notes :fire:

Reply:
Just to clarify, are these fine for the beta release notes? Or should I flame them now?

Reply:
🔥